src/Microsoft.AspNetCore.SignalR.Core/DefaultHubLifetimeManager.cs

```diff
@@ -69,18 +69,37 @@ public override Task RemoveGroupAsync(string connectionId, string groupName)
 
         public override Task SendAllAsync(string methodName, object[] args)
         {
-            return SendAllWhere(methodName, args, c => true);
-        }
+            List<Task> tasks = null;
+            var message = CreateInvocationMessage(methodName, args);
 
-        private Task SendAllWhere(string methodName, object[] args, Func<HubConnectionContext, bool> include)
-        {
-            var count = _connections.Count;
-            if (count == 0)
+            foreach (var connection in _connections)
+            {
+                var task = connection.WriteAsync(message);
+
+                if (!task.IsCompleted)
```

Comment:
I think you want `IsCompletedSuccessfully`. It was added in netcoreapp2.0

Helper method from Json.NET:
```
        public static bool IsCompletedSucessfully(this Task task)
        {
            // IsCompletedSucessfully is the faster method, but only currently exposed on .NET Core 2.0
#if NETCOREAPP2_0
            return task.IsCompletedSucessfully;
#else
            return task.Status == TaskStatus.RanToCompletion;
#endif
        }
```